Find a functionally equivalent code:android.widget.CalendarView.getMinDate:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the minimal date supported by this {@link CalendarView} in milliseconds
 * since January 1, 1970 00:00:00 in {@link TimeZone#getDefault()} time
 * zone.
 * <p>
 * Note: The default minimal date is 01/01/1900.
 * <p>
 *
 * @return The minimal supported date.
 *
 * @attr ref android.R.styleable#CalendarView_minDate
 */

@@ -1,3 +1,3 @@
 {
-    return mMinDate.getTimeInMillis();
+    return mDelegate.getMinDate();
 }

